core/src/rpc_subscriptions.rs

```diff
@@ -80,11 +84,7 @@ fn add_subscription<K, S>(
 {
     let sink = subscriber.assign_id(sub_id.clone()).unwrap();
     let confirmations = confirmations.unwrap_or(0);
-    let confirmations = if confirmations > MAX_LOCKOUT_HISTORY {
-        MAX_LOCKOUT_HISTORY
-    } else {
-        confirmations
-    };
+    let confirmations = min(confirmations, MAX_LOCKOUT_HISTORY);
```

Comment:
oh! this was a bug I think. Shouldn't it be possible to subscribe to fully confirmed (finalized) accounts / signatures? I think it should be `min(confirmations, MAX_LOCKOUT_HISTORY + 1);